charts/backingservices/charts/constellation-messaging/templates/messaging-deployment.yaml

```diff
@@ -32,10 +41,28 @@ spec:
           {{ toYaml .Values.securityContext | nindent 10 }}
           {{- end }}
         args:
-          - --max-semi-space-size=1024
-          - port={{ .Values.pegaMessagingTargetPort }}
-          - path=/c11n-messaging
+        - --max-semi-space-size=1024
+        - port={{ $srvTargetPort }}
+        - path={{ .Values.urlPath }}
```

Comment:
@alemax22  Could you please update the path to urlPath similar to constellation deployment, like : urlPath={{ .Values.urlPath }} ?